homeassistant/components/switch/__init__.py

```diff
@@ -47,6 +48,7 @@
 PROP_TO_ATTR = {
     'current_power_mwh': ATTR_CURRENT_POWER_MWH,
     'today_power_mw': ATTR_TODAY_MWH,
+    'current_power_watt' : ATTR_CURRENT_POWER_WATT,
```

Comment:
This will break all other switches because the property is not generally available. 

I don't to add this new general property. We chose to expose current power in mwh. If you want to add the current power in watts, use the device_state_attributes property